server/src/entity/User.ts

```diff
@@ -30,14 +32,19 @@ class User {
 
   @Field()
   @Column({ length: 255, type: "varchar" })
-  password: string;
+  hashedPassword: string;
 
   @Field()
   @Column({ length: 2083, type: "varchar" })
   picture: string;
 
   @Field()
-  @Column({ length: 20, type: "varchar", default: "user" })
+  @Column({
```

Comment:
can you delete this code, which is not relevant on this branch (but will be on the feature-role-management branch)?